app/models/user.rb

```diff
@@ -1,4 +1,5 @@
 class User < ApplicationRecord
+  alias_attribute :name, :email
```

Comment:
also present in https://github.com/dradis/dradis-ce/pull/498#pullrequestreview-287401164 (whit extra white line)

Reply:
I'm not convinced about this @brianp @dormi 

We've a problem w/ :name vs :email discrepancy that we need to address or keep making worse (like with this change).

Before we had issues with Comments and using the "login" in CE, but the "email" in Pro for @- mentions.

I think it's worth deciding here instead of just adding to the spaghetti ball.

Reply:
It was "simple" enough to manage without the alias. A couple tests need to change, but I think functionally everything else I've seen falls into place.
I'll do another pass to see if anything goes wrong using email instead of name.